Looms.PoS/Looms.PoS.Application/Utilities/Validators/FluentValidationExtensions.cs

```diff
@@ -10,4 +10,37 @@ public static IRuleBuilder<T, string> MustBeValidGuid<T>(this IRuleBuilder<T, st
                       .Must(id => Guid.TryParse(id, out _) && id.Length == 36)
                       .WithMessage("{PropertyName} is not a valid Guid");
     }
+
+    public static IRuleBuilder<T, string> MustBeValidDateTime<T>(this IRuleBuilder<T, string> builder)
+    {
+        return builder.NotEmpty()
+                      .Must(date =>
+                      {
+                          try
+                          {
+                              DateTimeHelper.ConvertToUtc(date);
+                              return true;
+                          }
+                          catch (FormatException)
+                          {
+                              return false;
+                          }
+                      })
+                      .WithMessage("{PropertyName} is not a valid DateTime");
+    }
+    public static IRuleBuilder<T, string> MustBeWithinBusinessHours<T>(this IRuleBuilder<T, string> builder)
+    {
+        int businessStartHour = 9;
```

Comment:
extract into separate file or make it a class property

Reply:
extracted to new class. file is BusinessHours.cs